tests/dummy/app/pods/application/template.hbs

```diff
@@ -1,5 +1,13 @@
 {{import Button from "ui/button"}}
 {{import LocalButton from "./button"}}
+{{import dirtyNamedGlobal from "ui/button"}}
+{{import dirtyNamedLocal from "./button"}}
+{{import Pseudo_Valid_Global from "ui/button"}}
+{{import Pseudo_Valid_Local from "./button"}}
 
 <Button>I'm a globally referenced button</Button>
-<LocalButton>I'm a locally referenced button</LocalButton>
\ No newline at end of file
+<LocalButton>I'm a locally referenced button</LocalButton>
+<dirtyNamedGlobal>I'm a dirty named global button</dirtyNamedGlobal>
```

Comment:
Could we change this to a slightly more descriptive name? I.e. `incorrectlyCamelCasedAbsoluteImport`?

Reply:
And perhaps `incorrectlyCamelCasedRelativeImport` below?

Reply:
done